test/integration/apiserver/admissionwebhook/match_conditions_test.go

```diff
@@ -289,35 +288,6 @@ func TestMatchConditions(t *testing.T) {
 				matchConditionsTestPod("test2", "default"),
 			},
 		},
-		{
-			name: "without strict cost enforcement: Authz check does not exceed per call limit",
```

Comment:
@aaron-prindle @Jefftree Should this test be retained but with the emulation version be set to 1.31/

Reply:
note that we could not reset the feature gate to false after locking default to true. Would emulation version still work in this case?

Reply:
If emulation version is at 1.31 we will emulate beta behavior which still has the default to true but allows the feature to be disabled. See https://github.com/kubernetes/kubernetes/pull/127093/files#diff-8868f5a770c851b2202de4360d037910dccf41dd67cfd213baf5e58eb8e4da88R2987 to allow disablement testing after promotion to GA. We should keep the disablement tests as best as we can.

Reply:
I guess the above setting only works for unit test since in integration test we would be using startTestServer to start the server which will set the emulation version to the default one: https://github.com/kubernetes/kubernetes/blob/3582dce11590577712f9a8db698d7e6897e23158/cmd/kube-apiserver/app/testing/testserver.go#L198

Reply:
Thanks for pointing this out, I'll work on a fix.

Reply:
Opened https://github.com/kubernetes/kubernetes/pull/127643